_assets/ci/Jenkinsfile.android

```diff
@@ -55,6 +55,16 @@ pipeline {
       }
     }
 
+    stage('Generate') {
+      environment {
+          /* Regenerate at least once */
+          GO_GENERATE_FAST_RECACHE = "true"
```

Comment:
Why not in global `environment` section like in `_assets/ci/Jenkinsfile.ios`? Seems cleaner.

Reply:
@jakubgs Further targets like `build` and `test` are dependant on the `generate` target.
I run it once as a separate stage with `RECACHE` option. But further the cache should be reused.

Reply:
If we use `GO_GENERATE_FAST_DIR = "${env.WORKSPACE_TMP}/go-generate-fast"` we get two birds stoned at the same time:

- We avoid weird bugs when parallel jobs share the same cache folder in user home.
- We make sure the cache is created fresh for every job, since `WORKSPACE_TMP` is always clean.